src/Makefile

```diff
@@ -118,6 +118,11 @@ uninstall:
 	test ! -f $(BINDIR)/atdgen.exe || rm $(BINDIR)/atdgen.exe
 	ocamlfind remove atdgen
 
+findlib-install:
+	ocamlfind install atdgen META \
+		 $(MLI) $(CMI) $(CMO) $(CMX) $(CMXS) $(O) \
+			atdgen.cma atdgen.a atdgen.cmxa
+
```

Comment:
it would also be useful for packaging in distros if that ocamlfind command line has a `$(OCAMLFIND_ARGS)` or similar to permit `-destdir` to be added from outside the build